lldb/include/lldb/Target/Process.h

```diff
@@ -2674,6 +2680,9 @@ void PruneThreadPlans();
                             const AddressRange &range, size_t alignment,
                             Status &error);
 
+  lldb::RunDirection GetBaseDirection() const { return m_base_direction; }
```

Comment:
These need some comments.  What is a "BaseDirection".  And how is it different from the "RunDirection"?

Reply:
Somewhere you should say what your basic strategy is.  This is one good place, or you can add a sequel to the novella in ThreadPlan.h if that's more appropriate.

Reply:
Add comments to `Process::GetBaseDirection()` and `Process::SetBaseDirection()`.
Added text to `ThreadPlan.h` novella.